Buddies/Controls/FilterSearchBar.swift

```diff
@@ -152,12 +275,16 @@ class FilterSearchBar : UISearchBar, UISearchBarDelegate {
         // Store request params #NoRaceConditions
         self.lastSearchParams = myParams
         
+        let start = DateRangeSliderDelegate.instance.getSliderDate(sliderValue: CGFloat(myParams.dateMin))
```

Comment:
Seems a bit wordy - would there be a way to clean this up? (Not quite sure how the architecture is working here)

Reply:
Definetly. I wasn't sure how this method was being used and didn't fix it. I think I'm going to change it to be called like this:

```swift
DateRangeSliderDelegate.getSliderDate(sliderIndex: myParams.dateMin)
```

I think that looks much better. Thanks for calling that out!

Reply:
cool!